src/redist/redist.csproj

```
@@ -6,6 +6,7 @@
     <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
     <ResolveAssemblyReferencesSilent>true</ResolveAssemblyReferencesSilent>
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>none</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
+    <BundleRuntimePacks Condition="'$(DotNetBuildFromSource)' == 'true'">true</BundleRuntimePacks>
```

I'd suggest the following if you want source build to be able to turn this off.

```suggestion
    <BundleRuntimePacks Condition="'$(BundleRuntimePacks)' == '' And '$(DotNetBuildFromSource)' == 'true'">true</BundleRuntimePacks>
```